GVFS/GVFS.UnitTests/Common/JsonTracerTests.cs

```diff
@@ -94,5 +94,23 @@ public void EventMetadataWithKeywordsIsOptional()
                 listener.EventNamesRead.ShouldContain(x => x.Equals("Error"));
             }
         }
+
+        [TestCase]
+        public void StartEventDoesNotDispatchTelemetry()
+        {
+            using (JsonTracer tracer = new JsonTracer("Microsoft-GVFS-Test", "StartEventDoesNotDispatchTelemetry", disableTelemetry: true))
+            using (MockListener listener = new MockListener(EventLevel.Verbose, Keywords.Telemetry))
+            {
+                tracer.AddEventListener(listener);
+
+                using (ITracer activity = tracer.StartActivity("TestActivity", EventLevel.Informational, Keywords.Telemetry, null))
+                {
+                    listener.EventNamesRead.ShouldBeEmpty();
+
+                    activity.Stop(null);
+                    listener.EventNamesRead.ShouldBeNonEmpty();

```

Comment:
nit: is there a specific event name we could look for?

Reply:
Yep, I made these more restrictive.